<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqCameraToolbar</class>
 <widget class="QToolBar" name="pqCameraToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Controls</string>
  </property>
  <property name="orientation">
   <enum>Qt::Horizontal</enum>
  </property>
  <action name="actionResetCamera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqResetCamera.png</normaloff>:/pqWidgets/Icons/pqResetCamera.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="iconText">
    <string>Reset Camera</string>
   </property>
   <property name="statusTip">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionPositiveX">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqXPlus.png</normaloff>:/pqWidgets/Icons/pqXPlus.png</iconset>
   </property>
   <property name="text">
    <string>+X</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to +X</string>
   </property>
   <property name="statusTip">
    <string>Set view direction to +X</string>
   </property>
   <property name="whatsThis">
    <string>Set view direction to +X</string>
   </property>
  </action>
  <action name="actionNegativeX">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqXMinus.png</normaloff>:/pqWidgets/Icons/pqXMinus.png</iconset>
   </property>
   <property name="text">
    <string>-X</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to -X</string>
   </property>
   <property name="statusTip">
    <string>Set view direction to -X</string>
   </property>
   <property name="whatsThis">
    <string>Set view direction to -X</string>
   </property>
  </action>
  <action name="actionPositiveY">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqYPlus.png</normaloff>:/pqWidgets/Icons/pqYPlus.png</iconset>
   </property>
   <property name="text">
    <string>+Y</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to +Y</string>
   </property>
   <property name="statusTip">
    <string>Set view direction to +Y</string>
   </property>
   <property name="whatsThis">
    <string>Set view direction to +Y</string>
   </property>
  </action>
  <action name="actionNegativeY">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqYMinus.png</normaloff>:/pqWidgets/Icons/pqYMinus.png</iconset>
   </property>
   <property name="text">
    <string>-Y</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to -Y</string>
   </property>
   <property name="statusTip">
    <string>Set view direction to -Y</string>
   </property>
   <property name="whatsThis">
    <string>Set view direction to -Y</string>
   </property>
  </action>
  <action name="actionPositiveZ">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqZPlus.png</normaloff>:/pqWidgets/Icons/pqZPlus.png</iconset>
   </property>
   <property name="text">
    <string>+Z</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to +Z</string>
   </property>
   <property name="statusTip">
    <string>Set view direction to +Z</string>
   </property>
   <property name="whatsThis">
    <string>Set view direction to +Z</string>
   </property>
  </action>
  <action name="actionNegativeZ">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqZMinus.png</normaloff>:/pqWidgets/Icons/pqZMinus.png</iconset>
   </property>
   <property name="text">
    <string>-Z</string>
   </property>
   <property name="toolTip">
    <string>Set view direction to -Z</string>
   </property>
   <property name="statusTip">
    <string>Set view direction to -Z</string>
   </property>
   <property name="whatsThis">
    <string>Set view direction to -Z</string>
   </property>
  </action>
  <action name="actionRotate90degCW">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqRotateCameraCW.png</normaloff>:/pqWidgets/Icons/pqRotateCameraCW.png</iconset>
   </property>
   <property name="text">
    <string></string>
   </property>
   <property name="toolTip">
    <string>Rotate 90° clockwise</string>
   </property>
   <property name="statusTip">
    <string>Rotate 90° clockwise</string>
   </property>
   <property name="whatsThis">
    <string>Rotate 90° clockwise</string>
   </property>
  </action>
  <action name="actionRotate90degCCW">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqRotateCameraCCW.png</normaloff>:/pqWidgets/Icons/pqRotateCameraCCW.png</iconset>
   </property>
   <property name="text">
    <string></string>
   </property>
   <property name="toolTip">
    <string>Rotate 90° counterclockwise</string>
   </property>
   <property name="statusTip">
    <string>Rotate 90° counterclockwise</string>
   </property>
   <property name="whatsThis">
    <string>Rotate 90° counterclockwise</string>
   </property>
  </action>
  <action name="actionZoomToBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqZoomToSelection.png</normaloff>:/pqWidgets/Icons/pqZoomToSelection.png</iconset>
   </property>
   <property name="text">
    <string>Zoom to Box</string>
   </property>
   <property name="statusTip">
    <string>Zoom to Box</string>
   </property>
  </action>
  <action name="actionZoomToData">
   <property name="text">
    <string>ZTD</string>
   </property>
   <property name="toolTip">
    <string>Zoom To Data</string>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqZoomToData.png</normaloff>:/pqWidgets/Icons/pqZoomToData.png</iconset>
   </property>
  </action>
  <addaction name="actionResetCamera"/>
  <addaction name="actionZoomToData"/>
  <addaction name="actionZoomToBox"/>
  <addaction name="actionPositiveX"/>
  <addaction name="actionNegativeX"/>
  <addaction name="actionPositiveY"/>
  <addaction name="actionNegativeY"/>
  <addaction name="actionPositiveZ"/>
  <addaction name="actionNegativeZ"/>
  <addaction name="actionRotate90degCW"/>
  <addaction name="actionRotate90degCCW"/>
 </widget>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
